packages/components/src/Radio/RadioGroup/RadioGroup.module.scss

```diff
@@ -0,0 +1,27 @@
+@import "~@kaizen/design-tokens/sass/color";
+@import "~@kaizen/design-tokens/sass/spacing";
+
+.radioGroupLabel {
+  margin-bottom: $spacing-sm;
```

Comment:
Should we update all spacing tokens to use `$spacing-{number}`equivalent in these PRs?

Reply:
Hm, nah leave them. We can fix them later when we transition to CSS Modules or something.